doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -1001,7 +1001,6 @@ table.docutils {
   margin-bottom: 1rem;
   line-height: 1rem;
   max-width: 100%;
-  display: block;
```

Comment:
This disallow centering the table (not a CSS expert). It does not have influence on the `overflow-x` that we fixed recently.